### DISCUSSION TITLE
live document

### DISCUSSION THREAD

**Discussion by Olajide811** [community/community#126045] [Programming Help]
### Select Topic Area

Question

### Body

please i need help i had live my html project but its showing blank page 
![blank page](https://github.com/community/community/assets/170964454/3db3a0b0-dab9-47c4-9110-670db6730f06)

**Comment by mprabhashmp** [comment]
Create a new repository on Github with the name yourusername.github.io, where yourusername is your Github username.
Upload your static website files to the repository. Your homepage should be named index.html.
Go to the repository's Settings and scroll down to the Github Pages section.
Select the main branch as the source for Github Pages and click Save.
Wait a few minutes for Github to build and deploy your website. You should see a message indicating that your site is published.

**Reply by Olajide811** [reply]
i just got done with 2 repository and i'm still getting a blank page

**Comment by Jayasurya5454** [comment]


**Reply by Olajide811** [reply]
Do you have any solution

**Comment by davevad93** [comment]
Hi @Olajide811 ,

I checked your repo, the problem is that your `index.html` is empty, you have to put your main `html` code there, otherwise it will not show anything.

**Comment by arashtabaa** [comment]
It looks like you're having trouble with your HTML project, which is currently showing a blank page. Let's go through some common issues and how to fix them:

1. **Check for File References:**
   Ensure all your HTML, CSS, and JavaScript files are correctly linked. The paths should be relative to your `index.html` or absolute URLs.

2. **Console Errors:**
   Open the browser's Developer Tools (usually F12 or right-click on the page and select "Inspect") and check the Console tab for any errors.

3. **File Names and Extensions:**
   Make sure all your file names and extensions are correct. For example, `index.html` should be `index.html` and not `index.htm`.

4. **GitHub Pages Configuration:**
   If you're using GitHub Pages, make sure your repository is correctly configured. The `index.html` file should be at the root of the repository or in the `gh-pages` branch.

5. **Push Changes:**
   Ensure you have pushed all changes to your GitHub repository. Sometimes, the changes might not reflect if they are not pushed correctly.

Let's start by checking your `index.html` file and other linked files. Can you upload the HTML file or provide the content here? This will help me identify any issues with your code.

**Reply by Olajide811** [reply]
i had to change the source to Github action before it work out

**Comment by Olajide811** [comment]
i change my source from main branch to Github Actions and i got my website live 
![Capture](https://github.com/community/community/assets/170964454/a21f7a5d-66cd-41a4-8a23-8348edc0b7e3)
